index.html

```diff
@@ -224,37 +224,26 @@ <h2>Multi-Post Stories</h2>
     </div>
   </section>
   <section class="contact">
-    <div class="contact-us">
-      <h2>CONTACT US</h2>
-    </div>
-    <!-- paragraph 3 -->
-    <div class="paragraph3">
-      <p>If you have an application you are <br>
+    <article>
+      <h2 class="contactus">CONTACT US</h2>
+      <p class="paragraph3">If you have an application you are <br>
         interested in developing, a feature that you <br>
         need built or a project that needs coding. <br>
         I’d love to help with it </p>
-    </div>
-    <ul class="buttom">
-      <li class="input-text">
-        input-text
-      </li>
-    </ul>
-
-    <ul class="buttom">
-      <li class="input-email">
-        jeremy@gmail.com
-      </li>
-    </ul>
-    <ul class="buttom">
-      <li class="input-comment">
-        input text here
-      </li>
-    </ul>
-    <button class="btn2">See project</button>
-    <!-- line4 -->
-    <hr>
-<!-- social media2 -->
-<div class="section-one-column2">
+    </article>
+    <form>
```

Comment:
- [ ] Please use the action and method attribute to integrate with [formspree service](https://formspree.io/html). You need to save user's data and make sure they can reach you through this service.
- [ ]  `action` value should be similar to `https://formspree.io/f/{form_id}`
- [ ]  `method` value equals `POST`

Reply:
done